gossip/client_test.go

```diff
@@ -95,7 +93,7 @@ func TestClientGossip(t *testing.T) {
 	client := newClient(remote.is.NodeAddr)
 	// Use an insecure context. We're talking to unix socket which are not in the certs.
```

Comment:
nothing odd about that. The built-in certs only have localhost (and a bunch of acceptance test stuff).
Try running this test in secure mode, and servername verification will fail with "certificate not valid for /tmp/unix-....."